MAUI/ListView/scrolling.md

```diff
@@ -183,52 +183,15 @@ public partial class MainPage : ContentPage
 
 ## Maintain the scroll position while updating ItemsSource at runtime
 
-The `SfListView` have scrolled to top automatically when changing the `ItemsSource` at runtime. However, you can maintain the same scrolled position by using the `ScrollY` value of `ListViewScrollView` from the [VisualContainer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.VisualContainer.html). After changing the `ItemsSource`, you can pass the `ScrollY` value to [ScrollTo](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ScrollTo_System_Double_System_Boolean_) method of `SfListView` and scroll back to the same position.
-
-For `Horizontal` orientation, use the `ScrollX` value of `ListViewScrollView`.
-
-By using [Reflection](https://docs.microsoft.com/en-us/dotnet/api/system.reflection?redirectedfrom=MSDN&view=net-5.0), get the value of `ScrollOwner` from `VisualContainer` and use it.
+The `SfListView` have scrolled to top automatically when changing the `ItemsSource` at runtime. However, you can maintain the same scrolled position by using `SfListView.CanMaintainScrollPosition` property.If you set `SfListView.CanMaintainScrollPosition` to true then on changing ItemsSource, the newly added ItemsSource will be loaded with the previous ItemsSource’s ScrollOffset.
```

Comment:
The `SfListView` has scrolled to the top automatically when changing the `ItemsSource` at runtime. However, you can maintain the same scrolled position using the `SfListView.CanMaintainScrollPosition` property. If you set `SfListView.CanMaintainScrollPosition` to true, then on changing ItemsSource, the newly added ItemsSource will be loaded with the previous ItemsSource’s ScrollOffset.

Reply:
resolved